From 306d50c04e17ae30a59455782773ce34884a603e Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Sat, 9 Nov 2024 03:58:55 +0800
Subject: [PATCH 2/2] GERRIT: loongarch: match vpx_quantize_b_lsx's signature
 with rtcd

The signature of vpx_quantize_b_lsx in the source code doesn't match the RTCD script.

Changing the signature in the source code to match the RTCD script.

Change-Id: I76012150d0954e51b21ec5a230f1ccf79ee90a2d

Link: https://chromium-review.googlesource.com/c/webm/libvpx/+/6008941
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 vpx_dsp/loongarch/quantize_lsx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vpx_dsp/loongarch/quantize_lsx.c b/vpx_dsp/loongarch/quantize_lsx.c
index 9bb1691e2..1299e75e9 100644
--- a/vpx_dsp/loongarch/quantize_lsx.c
+++ b/vpx_dsp/loongarch/quantize_lsx.c
@@ -91,7 +91,7 @@ static INLINE int16_t accumulate_eob(__m128i eob) {
 
 #if !CONFIG_VP9_HIGHBITDEPTH
 
-void vpx_quantize_b_lsx(const int16_t *coeff_ptr, intptr_t n_coeffs,
+void vpx_quantize_b_lsx(const tran_low_t *coeff_ptr, intptr_t n_coeffs,
                         const struct macroblock_plane *const mb_plane,
                         tran_low_t *qcoeff_ptr, tran_low_t *dqcoeff_ptr,
                         const int16_t *dequant_ptr, uint16_t *eob_ptr,
-- 
2.47.0.windows.2

